src/pages/graphql/schema/orders/mutations/request-guest-order-cancel.md

```diff
@@ -0,0 +1,69 @@
+---
+title: requestGuestOrderCancel mutation
+---
+
+# requestGuestOrderCancel mutation
+
+The `requestGuestOrderCancel` mutation allows a guest to request for cancelling an order, passing the order token and a cancellation reason.
+
```

Comment:
```suggestion
The `requestGuestOrderCancel` mutation allows a guest customer to request for cancelling an order, passing the order token and a cancellation reason.

```
I would also change all occurences of `guest` to `guest customer` for consistency.